sdk/storage/sample/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.12)
+
+set(TARGET_NAME "azure-storage-sample")
```

Comment:
`TARGET_NAME` is a quite common name, i'm worried it will inadvertently override some variable. Can we not use it and just write `azure-storage-sample` in `add_executable` and `target_link_libraries` commands?